src/Orchard.Web/Modules/Orchard.Autoroute/Migrations.cs

```diff
@@ -49,5 +62,46 @@ public class Migrations : DataMigrationImpl {
 
             return 4;
         }
+
+        public int UpdateFrom4() {
+            // Adding some culture neutral patterns if they don't exist
+            var types = _contentManager.GetContentTypeDefinitions().Where(t => t.Parts.Any(p => p.PartDefinition.Name.Equals(typeof(AutoroutePart).Name)));
+            foreach (var type in types) {
+                var typeDefinition = ContentDefinitionManager.GetTypeDefinition((type.Name));
+                if (typeDefinition != null) {
+                    var settings = typeDefinition.Parts.First(x => x.PartDefinition.Name == "AutoroutePart").Settings.GetModel<AutorouteSettings>();
+                    if (!settings.Patterns.Any(x => String.IsNullOrWhiteSpace(x.Culture))) {
+                        string siteCulture = _cultureManager.GetSiteCulture();
+                        string newPatterns = "";
```

Comment:
Make `newPatterns` a `List<string>` then at the end you can just join all of them using `String.Join(", ")`. This way you don't have to deal with concatenating `,` and then having to remove the last one.